curriculum/structure/blocks/workshop-major-browsers-list.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Build a List of Major Web Browsers",
+  "isUpcomingChange": true,
+  "dashedName": "workshop-major-browsers-list",
+  "helpCategory": "HTML-CSS",
+  "blockLayout": "challenge-grid",
+  "challengeOrder": [
+    { "id": "68e6698b29e562d21b1d260c", "title": "Step 1" },
+    { "id": "68ecfa0f03d88990bb96a33c", "title": "Step 2" },
+    { "id": "68ecfa1003d88990bb96a33d", "title": "Step 3" },
+    { "id": "68ecfa1003d88990bb96a33e", "title": "Step 4" },
+    { "id": "68ecfa1003d88990bb96a33f", "title": "Step 5" }
+  ],
+  "blockType": "workshop",
```

```suggestion
  "blockLabel": "workshop",
```

per https://github.com/freeCodeCamp/freeCodeCamp/pull/62548